docs/RFPs/Open/IDE_for_ink!_Smart_Contracts.md

```diff
@@ -0,0 +1,22 @@
+# Browser based IDE for ink! Smart Contracts 
+
+* **Status:** Open
+* **Proposer:** [David Hawig](https://github.com/Noc2)
+
+## Project Description :page_facing_up:  
+
+[ink!](https://github.com/paritytech/ink) is a domain-specific language for writing smart contracts in Rust and compiles to Wasm code. You can deploy ink! contracts on parachains that support the contracts pallet, as well as on stand-alone blockchains built with Substrate.
+
+The goal of this RFP is to find teams that would like to develop or maintain a browser based playground for editing, sharing & compiling ink! Smart Contracts. We are hereby specifically looking for a developer or team that wants to continue to work on the [ink! Playground](https://github.com/paritytech/ink-playground). The [ink! Playground](https://ink-playground.substrate.io/), previously maintained by Parity, currently utilizes Typescript, React, Docker, and [Monaco Editor](https://microsoft.github.io/monaco-editor/). Teams could stick with this tech stack, or choose to build something new if desired. For example, there has been some discussion around the viability of implementing GitHub Codespaces in combination with GitHub Actions and previously teams developed a [Remix plugin](https://github.com/blockchain-it-hr/ink-remix-plugin).
```

Comment:
```suggestion
The goal of this RFP is to find teams that would like to develop or maintain a browser-based playground for editing, compiling & sharing ink! smart contracts. The [ink! Playground](https://ink-playground.substrate.io/), currently maintained by Parity, utilizes Typescript, React, Docker, and [Monaco Editor](https://microsoft.github.io/monaco-editor/). Teams could continue to work on the [ink! Playground](https://github.com/paritytech/ink-playground), or choose to build something new if desired. For example, it might be possible to leverage GitHub Codespaces in combination with GitHub Actions or the previously developed [ink! Remix plugin](https://github.com/blockchain-it-hr/ink-remix-plugin).
```

Reply:
The ink playground is no longer maintained by parity (since 3 weeks), regarding "currently maintained by Parity"

Reply:
Changed that back